templates/communitypage.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="../static/css/style_communitypage.css">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+    <!-- FontAwesome -->
+    <link rel="stylesheet" href= "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
+    <title>Blogger | Community Page</title>
+</head>
+<body>
+<!-- Navbar starts here -->
```

Comment:
Hey @AdityaAshvin can we please make a new file for header so that we can use it in every page.

Reply:
@Kajol-Kumari Should i create a base.html file with the header and  footer in that?

Reply:
No, just make a separate html file for header i.e `header.html` and for footer `footer.html` and just render it where ever required like in community page and other pages like home page.